modules/local/remove_unknown_regions/tests/main.nf.test

```diff
@@ -23,11 +23,14 @@ nextflow_process {
 
         then {
             assert process.success
-            assert snapshot(process.out).match()
+            assert snapshot(
+                process.out.bed,
+                ['versions', process.out.versions]
+            ).match()
```

Comment:
why not just `process.out` here? it's still pretty clear like that IMO

Reply:
I'd rather be explicit, that way no numbered output